### ISSUE TITLE
False Positives for components with default imports?

### ISSUE THREAD

**Issue by Bewinxed** [pheuter/svelte-check-rs#80]
Hey mate, Love this project, so so much faster than svelte-check, but i'm getting a lot of false-positives, here's the log:

for example 
`	import Icon from '@iconify/svelte';`
then used inline, and some other types like `chrome.xxx` namespace showing as  unknown but in lsp it shows fine, any ideas?

```
extension/entrypoints/popup/App.svelte:228:5
Warning: A11y: Avoid using autofocus (a11y-autofocus)

src/routes/(app)/focus/+page.svelte:23:37
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:23:90
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:24:40
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:25:38
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:26:37
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:27:34
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:28:39
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:55:23
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:56:24
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/routes/(app)/focus/+page.svelte:612:5
Warning: A11y: Heading levels should not be skipped (h1 followed by h3) (a11y-structure)

src/routes/(app)/boards/[boardId]/+page.svelte:121:10
Warning: A11y: Heading levels should not be skipped (h1 followed by h3) (a11y-structure)

src/routes/(app)/home/+page.svelte:27:44
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/notifications/+page.svelte:9:31
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/notifications/+page.svelte:10:28
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:29:25
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:30:24
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:31:35
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:32:36
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:33:34
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:36:37
Warning: This reference only captures the initial value of `data`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/routes/(app)/settings/+page.svelte:232:8
Warning: A11y: Heading levels should not be skipped (h2 followed by h4) (a11y-structure)

src/routes/(app)/browse/+page.svelte:218:5
Warning: A11y: Heading levels should not be skipped (h1 followed by h3) (a11y-structure)

src/routes/onboarding/+page.svelte:97:1
Warning: A11y: <main> has an implicit role of 'main'. Avoid redundant role (a11y-no-redundant-roles)

src/lib/components/cards/VisualNoteCard.svelte:96:1
Warning: A11y: <article> has an implicit role of 'article'. Avoid redundant role (a11y-no-redundant-roles)

src/lib/components/focus/FocusSummaryModal.svelte:119:3
Warning: A11y: <div> with click handler must have a role attribute (a11y-no-static-element-interactions)

src/lib/components/celebration/LevelUpModal.svelte:59:3
Warning: A11y: <div> with click handler must have a role attribute (a11y-no-static-element-interactions)

src/lib/components/reader/ChatPanel.svelte:130:2
Warning: A11y: Click events must be accompanied by a keyboard handler (onkeydown or onkeyup) (a11y-click-events-have-key-events)

src/lib/components/ui/toggle-group/toggle-group.svelte:36:3
Warning: This reference only captures the initial value of `variant`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/lib/components/ui/toggle-group/toggle-group.svelte:37:3
Warning: This reference only captures the initial value of `size`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/lib/components/ui/toggle-group/toggle-group.svelte:38:3
Warning: This reference only captures the initial value of `spacing`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/lib/components/ui/input-group/input-group-addon.svelte:40:1
Warning: A11y: Click events must be accompanied by a keyboard handler (onkeydown or onkeyup) (a11y-click-events-have-key-events)

src/lib/components/ui/chart/chart-container.svelte:20:27
Warning: This reference only captures the initial value of `id`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/lib/components/ui/input-otp/input-otp-separator.svelte:13:1
Warning: A11y: Elements with role='separator' must have 'aria-valuenow' (a11y-role-has-required-aria-props)

src/lib/components/ui/input-otp/input-otp-separator.svelte:13:1
Warning: A11y: Elements with role='separator' must have 'aria-valuemax' (a11y-role-has-required-aria-props)

src/lib/components/ui/input-otp/input-otp-separator.svelte:13:1
Warning: A11y: Elements with role='separator' must have 'aria-valuemin' (a11y-role-has-required-aria-props)

src/lib/components/ui/carousel/carousel.svelte:25:3
Warning: This reference only captures the initial value of `orientation`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/ui/carousel/carousel.svelte:29:12
Warning: This reference only captures the initial value of `opts`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/ui/carousel/carousel.svelte:30:3
Warning: This reference only captures the initial value of `plugins`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/pomodoro/PomodoroTimer.svelte:33:29
Warning: This reference only captures the initial value of `workMinutes`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/pomodoro/ReadingTargetCard.svelte:43:28
Warning: This reference only captures the initial value of `target`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/pomodoro/ReadingTargetCard.svelte:155:5
Warning: A11y: <label> must have an associated control (use 'for' attribute or nest a control) (a11y-label-has-associated-control)

src/lib/components/boards/BoardCard.svelte:110:1
Warning: A11y: <article> has an implicit role of 'article'. Avoid redundant role (a11y-no-redundant-roles)

src/lib/components/boards/BoardCard.svelte:110:1
Warning: A11y: Non-interactive elements like <article> should not have tabindex (a11y-no-noninteractive-tabindex)

src/lib/components/boards/BoardsPage.svelte:45:50
Warning: This reference only captures the initial value of `initialSuggested`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/boards/BoardsPage.svelte:46:49
Warning: This reference only captures the initial value of `initialAccepted`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/stale/StaleContentCard.svelte:34:63
Warning: This reference only captures the initial value of `resurfacing`. Did you mean to reference it inside a closure instead? (state-referenced-locally)

src/lib/components/stale/StaleContentCard.svelte:77:1
Warning: A11y: <article> has an implicit role of 'article'. Avoid redundant role (a11y-no-redundant-roles)

src/lib/components/tags/TagManager.svelte:310:5
Warning: A11y: <label> must have an associated control (use 'for' attribute or nest a control) (a11y-label-has-associated-control)

src/lib/components/tags/TagManager.svelte:335:6
Warning: A11y: <label> must have an associated control (use 'for' attribute or nest a control) (a11y-label-has-associated-control)

src/lib/components/tags/TagManager.svelte:374:5
Warning: A11y: <label> must have an associated control (use 'for' attribute or nest a control) (a11y-label-has-associated-control)

src/lib/components/tags/TagManager.svelte:399:6
Warning: A11y: <label> must have an associated control (use 'for' attribute or nest a control) (a11y-label-has-associated-control)

src/lib/components/home/HeroCard.svelte:45:45
Warning: This reference only captures the initial value of `currentMood`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/reactions/ReactionBar.svelte:49:55
Warning: This reference only captures the initial value of `currentReaction`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/avatar/AvatarEditor.svelte:74:48
Warning: This reference only captures the initial value of `avatar`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/wikilinks/BacklinksSection.svelte:49:26
Warning: This reference only captures the initial value of `initialBacklinks`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/wikilinks/BacklinksSection.svelte:51:42
Warning: This reference only captures the initial value of `initialBacklinks`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/wikilinks/BacklinksSection.svelte:52:21
Warning: This reference only captures the initial value of `initialCount`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

src/lib/components/wikilinks/BacklinksSection.svelte:53:27
Warning: This reference only captures the initial value of `startCollapsed`. Did you mean to reference it inside a derived instead? (state-referenced-locally)

extension/lib/capture.ts:65:47
Error: Cannot find namespace 'chrome'. (ts(TS2503))

extension/lib/capture.ts:88:27
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/capture.ts:99:24
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/capture.ts:111:24
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/capture.ts:118:44
Error: Cannot find namespace 'chrome'. (ts(TS2503))

extension/lib/capture.ts:142:42
Error: Cannot find namespace 'chrome'. (ts(TS2503))

extension/lib/capture.ts:154:7
Error: Cannot find namespace 'chrome'. (ts(TS2503))

extension/lib/plugins/twitter/index.ts:22:26
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/plugins/twitter/index.ts:23:37
Error: Parameter 'c' implicitly has an 'any' type. (ts(TS7006))

extension/lib/plugins/twitter/index.ts:28:27
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/plugins/twitter/index.ts:29:38
Error: Parameter 'c' implicitly has an 'any' type. (ts(TS7006))

extension/lib/plugins/twitter/index.ts:42:26
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/plugins/twitter/index.ts:43:36
Error: Parameter 'c' implicitly has an 'any' type. (ts(TS7006))

extension/lib/plugins/twitter/index.ts:47:28
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/lib/plugins/twitter/index.ts:48:33
Error: Parameter 'c' implicitly has an 'any' type. (ts(TS7006))

extension/entrypoints/popup/App.svelte:40:3
Error: Cannot find name 'browser'. (ts(TS2304))

extension/entrypoints/popup/App.svelte:40:42
Error: Parameter 'message' implicitly has an 'any' type. (ts(TS7006))

extension/entrypoints/popup/App.svelte:122:9
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/entrypoints/popup/App.svelte:128:27
Error: Cannot find name 'browser'. (ts(TS2304))

extension/entrypoints/popup/App.svelte:143:27
Error: Cannot find name 'browser'. (ts(TS2304))

extension/entrypoints/popup/App.svelte:181:3
Error: Cannot find name 'chrome'. (ts(TS2304))

extension/entrypoints/popup/App.svelte:198:10
Error: Cannot find name 'chrome'. (ts(TS2304))

src/lib/components/boards/BoardCard.svelte:191:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/boards/BoardCard.svelte:193:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/boards/BoardCard.svelte:204:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/boards/BoardsPage.svelte:249:8
Error: Type '(board: BoardSuggestion) => Promise<void>' is not assignable to type '(board: BoardSuggestion) => void'. (ts(TS2322))

src/lib/components/boards/BoardsPage.svelte:250:8
Error: Type '(board: BoardSuggestion) => void' is not assignable to type '(board: BoardSuggestion) => void'. Two different types with this name exist, but they are unrelated. (ts(TS2719))

src/lib/components/boards/BoardsPage.svelte:251:8
Error: Type '(board: BoardSuggestion) => void' is not assignable to type '(board: BoardSuggestion) => void'. Two different types with this name exist, but they are unrelated. (ts(TS2719))

src/lib/components/boards/BoardsPage.svelte:271:8
Error: Type '(board: BoardSuggestion) => void' is not assignable to type '(board: BoardSuggestion) => void'. Two different types with this name exist, but they are unrelated. (ts(TS2719))

src/lib/components/boards/SidebarBoards.svelte:58:12
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/boards/SidebarBoards.svelte:62:12
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/browse/MapView.svelte:100:47
Error: No overload matches this call. (ts(TS2769))

src/lib/components/browse/MapView.svelte:242:16
Error: This expression is not callable. (ts(TS2349))

src/lib/components/celebration/ArchiveToast.svelte:24:4
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/celebration/LevelUpModal.svelte:24:19
Error: 'ConfettiBurst' refers to a value, but is being used as a type here. Did you mean 'typeof ConfettiBurst'? (ts(TS2749))

src/lib/components/celebration/LevelUpModal.svelte:83:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/celebration/LevelUpModal.svelte:103:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:63:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:65:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:88:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:91:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:126:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/ArticleTrackingIndicator.svelte:137:4
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/FocusSummaryModal.svelte:42:19
Error: 'ConfettiBurst' refers to a value, but is being used as a type here. Did you mean 'typeof ConfettiBurst'? (ts(TS2749))

src/lib/components/focus/FocusSummaryModal.svelte:132:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/FocusSummaryModal.svelte:157:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/FocusSummaryModal.svelte:168:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/FocusSummaryModal.svelte:187:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/focus/FocusSummaryModal.svelte:213:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/HeroCard.svelte:138:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/HeroCard.svelte:157:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:173:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:223:12
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:234:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:251:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:253:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:265:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:278:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:280:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:293:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:309:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/home/PickCard.svelte:316:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:180:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:188:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:253:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:263:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:275:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/PomodoroTimer.svelte:285:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:105:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:112:6
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:122:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:196:30
Error: Argument of type '{ onValueChange: (value: string) => void; children: () => { '{@render ...} must be called with a Snippet': "import type { Snippet } from 'svelte'"; } & unique symbol; }' is not assignable to parameter of type '__SvelteProps_Select_ & __SvelteCssProps'. (ts(TS2345))

src/lib/components/pomodoro/ReadingTargetCard.svelte:162:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:174:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:216:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/pomodoro/ReadingTargetCard.svelte:223:5
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/reader/ArticleReader.svelte:98:22
Error: Cannot find namespace 'DOMPurify'. (ts(TS2503))

src/lib/components/reader/ReaderContent.svelte:31:17
Error: Cannot find namespace 'DOMPurify'. (ts(TS2503))

src/lib/components/reader/ReaderContent.svelte:93:30
Error: Argument of type 'string | TrustedHTML' is not assignable to parameter of type 'string'. (ts(TS2345))

src/lib/components/reader/ReaderContent.svelte:96:45
Error: Property 'slice' does not exist on type 'string | TrustedHTML'. (ts(TS2339))

src/lib/components/reader/ReaderContent.svelte:124:36
Error: Property 'length' does not exist on type 'string | TrustedHTML'. (ts(TS2339))

src/lib/components/reader/ReaderContent.svelte:125:39
Error: Property 'slice' does not exist on type 'string | TrustedHTML'. (ts(TS2339))

src/lib/components/reader/ReaderContent.svelte:138:40
Error: Property 'split' does not exist on type 'string | TrustedHTML'. (ts(TS2339))

src/lib/components/reader/ReaderContent.svelte:139:24
Error: Parameter 'text' implicitly has an 'any' type. (ts(TS7006))

src/lib/components/reader/ReaderContent.svelte:139:30
Error: Parameter 'i' implicitly has an 'any' type. (ts(TS7006))

src/lib/components/reader/ReaderContent.svelte:154:5
Error: Type 'string | TrustedHTML' is not assignable to type 'string'. (ts(TS2322))

src/lib/components/tags/IconPicker.svelte:174:4
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/tags/IconPicker.svelte:197:7
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/tags/IconPicker.svelte:220:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/tags/TagInput.svelte:224:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/tags/TagPill.svelte:85:3
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/components/ui/chart/chart-container.svelte:11:3
Error: Type 'unknown' is not assignable to type 'string | null'. (ts(TS2322))

src/lib/components/ui/chart/chart-container.svelte:20:33
Error: 'uid' is of type 'unknown'. (ts(TS18046))

src/lib/components/ui/sidebar/sidebar-rail.svelte:21:2
Error: Object literal may only specify known properties, but 'tabIndex' does not exist in type 'HTMLButtonAttributes & HTMLAttributes<any> & { onabort?: EventHandler<Event, HTMLButtonElement> | null | undefined; ... 107 more ...; onwheel?: WheelEventHandler<...> | ... 1 more ... | undefined; }'. Did you mean to write 'tabindex'? (ts(TS2561))

src/lib/components/wikilinks/WikiLink.svelte:78:21
Error: Object literal may only specify known properties, but 'asChild' does not exist in type '{ [x: symbol]: false | Attachment<HTMLAnchorElement> | null | undefined; [x: `data-${string}`]: any; 'on:copy'?: ClipboardEventHandler<HTMLAnchorElement> | null | undefined; ... 440 more ...; ref?: HTMLElement | ... 1 more ... | undefined; } & __SvelteCssProps'. Did you mean to write 'child'? (ts(TS2561))

src/routes/api/audio/reader-with-timestamps/+server.ts:188:4
Error: Type '"nPczCjzI2devNBz1zQrb"' is not assignable to type 'VoiceId | undefined'. (ts(TS2322))

src/routes/api/audio/reader/+server.ts:182:7
Error: Type '"nPczCjzI2devNBz1zQrb"' is not assignable to type 'VoiceId | undefined'. (ts(TS2322))

src/routes/api/audio/reader/+server.ts:255:31
Error: Property 'authorName' is missing in type '{ id: number; title: string | null; longSummary: string | null; shortSummary: string | null; content: string | null; }' but required in type '{ title: string | null; authorName: string | null; longSummary: string | null; shortSummary: string | null; content: string | null; }'. (ts(TS2741))

src/routes/api/chat/+server.ts:9:34
Error: Cannot find module '$lib/server/ai/openrouter' or its corresponding type declarations. (ts(TS2307))

src/routes/api/email/preferences/+server.ts:76:3
Error: Type 'string[] | undefined' is not assignable to type '("achievement_unlocked" | "buy_reminder" | "content_ready" | "daily_digest" | "elara_checkin" | "re_engagement" | "urgent_deadline" | "weekly_recap" | "welcome")[] | undefined'. (ts(TS2322))

src/routes/api/wikilinks/backlinks/+server.ts:16:31
Error: Property 'auth' does not exist on type 'Locals'. (ts(TS2339))

src/routes/api/wikilinks/backlinks/index/+server.ts:20:31
Error: Property 'auth' does not exist on type 'Locals'. (ts(TS2339))

src/routes/auth/login/+page.svelte:71:23
Error: '__each_0' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/login/+page.svelte:93:23
Error: '__each_1' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/login/+page.svelte:100:23
Error: '__each_2' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/setup/+page.svelte:71:23
Error: '__each_0' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/setup/+page.svelte:108:23
Error: '__each_1' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/setup/+page.svelte:130:23
Error: '__each_2' is possibly 'undefined'. (ts(TS18048))

src/routes/auth/setup/+page.svelte:137:23
Error: '__each_3' is possibly 'undefined'. (ts(TS18048))

src/routes/onboarding/+page.svelte:31:79
Error: No overload matches this call. (ts(TS2769))

src/routes/onboarding/+page.svelte:107:17
Error: This expression is not callable. (ts(TS2349))

src/routes/onboarding/+page.svelte:152:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:166:11
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:202:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:221:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:229:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:239:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:260:12
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:263:12
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:286:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:296:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:305:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:316:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:330:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:344:10
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:360:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:370:8
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/routes/onboarding/+page.svelte:382:9
Error: Value of type 'typeof Icon' is not callable. Did you mean to include 'new'? (ts(TS2348))

src/lib/server/services/signalService.ts:124:7
Error: This comparison appears to be unintentional because the types '-0.2 | 0.1 | 0.3 | 0.5' and '0' have no overlap. (ts(TS2367))

src/lib/server/services/signalService.ts:161:7
Error: This comparison appears to be unintentional because the types '-0.2 | 0.1 | 0.3 | 0.5' and '0' have no overlap. (ts(TS2367))

src/lib/server/services/staleContentService.ts:275:5
Error: Object literal may only specify known properties, and 'apiKey' does not exist in type '{ model?: string | undefined; maxTokens?: number | undefined; temperature?: number | undefined; }'. (ts(TS2353))

====================================
svelte-check-rs found 131 errors and 59 warnings in 418 files
```
